### DISCUSSION TITLE
OMG Extruder Question

### DISCUSSION THREAD

**Discussion by Marauder01** [Jyers/Marlin#1737] [General Discussion]
I've installed the OMG dual gear extruder in the Ender 3 V2 with it running Jyers, Great firmware BTW and thank you so much for building it.  I do have a question: would it possible to add into the menu the ability to reverse the extruder direction without having to re-flash the firmware all the time? 

Also being able to change the mesh setting from the menu from 3 to 5 would be great.

In the mean time what where is the seeing to reveres the extruder direction with in the firmware?
Thanks again
MC

**Comment by Marauder01** [comment]
OK I'm totally lost, I've edited and installed other Marlin software on my Ender 3 but I cannot seem to get this to compile correctly using platformio.  All I need to do is invert the extruder on the BL Touch 3x3 file.  when I go to compile it fails every time.  Anyone have a better editor and compiler?
Thanks

**Reply by Cal-LF** [reply]
@Marauder01 Are you using the E3V2 Template config files? They are where you need to start. Copy the configs closest to the one you want to the Marlin folder and do a compile with the least changes for a trial.
